js_modules/dagster-ui/packages/ui-core/src/instance/backfill/BackfillOpJobPartitionsTable.tsx

```diff
@@ -78,3 +84,69 @@ export const BackfillOpJobPartitionsTable = ({
     </Table>
   );
 };
+
+export const BackfillOpJobTarget = ({
+  backfill,
+  repoAddress,
+}: {
+  backfill: Pick<BackfillTableFragment, 'partitionSet' | 'partitionSetName'>;
+  repoAddress: RepoAddress | null;
+}) => {
+  const repo = useRepository(repoAddress);
+  const {partitionSet, partitionSetName} = backfill;
+
+  const buildHeader = () => {
+    if (partitionSet && repo) {
+      const link = workspacePipelinePath({
+        repoName: partitionSet.repositoryOrigin.repositoryName,
+        repoLocation: partitionSet.repositoryOrigin.repositoryLocationName,
+        pipelineName: partitionSet.pipelineName,
+        isJob: isThisThingAJob(repo, partitionSet.pipelineName),
+        path: `/partitions?partitionSet=${encodeURIComponent(partitionSet.name)}`,
+      });
+      return (
+        <Link style={{fontWeight: 500}} to={link}>
+          {partitionSet.name}
+        </Link>
+      );
+    }
+    if (partitionSetName) {
+      return <span style={{fontWeight: 500}}>{partitionSetName}</span>;
+    }
+    return null;
+  };
+
+  const buildRepoLink = () =>
+    repoAddress ? (
+      <Box flex={{direction: 'row', gap: 8, alignItems: 'center'}} style={{fontSize: '12px'}}>
+        <Icon name="repo" color={Colors.textLight()} />
+        <Link to={workspacePathFromAddress(repoAddress)}>
+          {repoAddressAsHumanString(repoAddress)}
+        </Link>
+      </Box>
+    ) : undefined;
```

Comment:
Could just be `const repoLink = repoAddress ? <...> : null;`

Reply:
Will change all these simple ones -- it seems like this was using the subfunctions pretty heavily